CMakeLists.txt

```diff
@@ -0,0 +1,67 @@
+# SPDX-FileCopyrightText: 2009 Cristian Oneț <onet.cristian@gmail.com>
+
+project(ofx)
+
+set (VERSION_MAJOR 0)
+set (VERSION_MINOR 10)
+set (VERSION_PATCH 5)
+set (VERSION ${VERSION_MAJOR}.${VERSION_MINOR}.${VERSION_PATCH})
+
+cmake_minimum_required(VERSION 2.6)
+
+list(APPEND CMAKE_MODULE_PATH  ${CMAKE_SOURCE_DIR})
+find_package(OpenSP REQUIRED)
+
+if(MSVC)
+    add_definitions(-D_CRT_SECURE_NO_WARNINGS)
+endif(MSVC)
+
+include_directories(
+    inc
+    ${OPENSP_INCLUDES}
+    ${CMAKE_INSTALL_PREFIX}/include
+)
+
+add_definitions(
+    -D__WIN32__
+    -DMAKEFILE_DTD_PATH="${CMAKE_PREFIX_PATH}/share/libofx/dtd/"
+)
+
+file(GLOB COEFF_HDRS inc/libofx.h lib/*.hh )
+file(GLOB SRC_SOURCES lib/*.cpp ${COEFF_HDRS})
+
+
+# We would like it to be shared, but libofx does not export symbols manually
+# thus it causes troubles on MSVC
+#add_library(ofx SHARED ${SRC_SOURCES})
+add_library(ofx ${SRC_SOURCES})
```

Comment:
I didn't write the comment about wanting it to be shared but that not working on MSVC, you did. If that's not the case, remove the comment.

Reply:
> you did.

Again, this is a leftover, and you yet again jump into conclusions here. All of this is a WIP, market Draft, so I would hope it's clear by now that assuming I wrote everything here is counterproductive.

Reply:
I usually do code reviews commit-by-commit for non-trivial PRs, it works better with Github than reviewing on the Files changed tab. If you don't want someone looking over your shoulder why show your work? You can `git rebase` away all of the messy stuff before making it public. You might also note that comments on code that you've changed has a gold Outdated label right at the top.

I'm not sure what any of that has to do with the spirit of Open Source contribution, though I'll stipulate for you that it may end up taking more time and that you might consider that wasted time.